cmd/bffctl/queue.go

```diff
@@ -2,12 +2,15 @@ package main
 
 import (
 	"fmt"
+	"strings"
+	"time"
+
 	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgtype"
+	"github.com/rs/xid"
 	"github.com/strideynet/bsky-furry-feed/store/gen"
 	"github.com/urfave/cli/v2"
 	"go.uber.org/zap"
-	"strings"
 )
 
 func queueCmd(log *zap.Logger, env *environment) *cli.Command {
```

Comment:
We can actually delete this command - I'd rather we started moving the CLI towards one-off operations like migrations. I'm always a little worried that if we keep something like the queue command around, we'll end up forgetting to adjust it in line w/ changes made in the API.